nixos/hardware/hetzner-online/default.nix

```
@@ -30,11 +30,9 @@
 
     networking.useNetworkd = true;
     networking.useDHCP = false;
-    # Hetzner servers commonly only have one interface, so its either to just match by that.
-    networking.usePredictableInterfaceNames = false;
 
     systemd.network.networks."10-uplink" = {
-      matchConfig.Name = "eth0";
+      matchConfig.Name = "eno1 ens1 enp1s0 eth0";
```

We can make this easier to override than:

```suggestion
      matchConfig.Name = lib.mkDefault "eno1 ens1 enp1s0 eth0";
```